src/Dto/DeletePostPayload.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Dto;
```

Comment:
I personally am used to `\Model` namespace
`\Dto`, for a demo, would make sens only if the dev reading know it (what we can imagine it is ok, but not sure :s)
wdyt?

Reply:
Model namespace is often used for database models (in Laravel for example), I think having Dto is less confusing

Reply:
In Doctrine, the database models are in the `Entity` namespace.